tests/bsim/bluetooth/audio/src/bap_broadcast_sink_test.c

```diff
@@ -26,8 +26,11 @@ CREATE_FLAG(flag_received);
 static struct bt_bap_broadcast_sink *g_sink;
 static struct bt_bap_stream broadcast_sink_streams[CONFIG_BT_BAP_BROADCAST_SNK_STREAM_COUNT];
 static struct bt_bap_stream *streams[ARRAY_SIZE(broadcast_sink_streams)];
-static struct bt_audio_codec_cap codec_cap_16_2_1 = BT_AUDIO_CODEC_LC3_CONFIG_16_2(
-	BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED);
+
+static const struct bt_audio_codec_cap codec_cap_16_2_1 = BT_AUDIO_CODEC_CAP_LC3(
```

Comment:
for 16_2_1, should this not be reflected in the inner params (e.g. fixed frequency)?